doc/tutorial.md

```diff
@@ -0,0 +1,212 @@
+      {
+      "name": "N",
+      "description": "Number of measurements."
+      }
+    ]
+```
+
+### **Step 5**: Defining the properties
+Now it is time to define the properties of our Entity. Here is where we can give meaning to our data. As for the dimensions, we add the properties as a list of `json` structures, each one having a **name**, **type**, **unit**, **description**, and if relevant, a list of **dimension**(s) (here called **dims**). Inserting the properties displayed in the table above, our Entity is complete and may look like
+
+```json
+{
+    "uri": "http://www.ontotrans.eu/0.1/solarPanelMeasurement",
+    "meta": "http://onto-ns.com/meta/0.3/EntitySchema",
+    "description": "Measurement data from one solar panel.",
+    "dimensions": [
+      {
+      "name": "N",
+      "description": "Number of measurements."
+      }
+    ],
+    "properties": [
+        {
+            "name":"t",
+            "type":"str",
+            "unit":"ns",
+            "dims": ["N"],
+            "description": "Time"
+        },
+
+        {
+            "name":"MPP",
+            "type":"float64",
+            "unit":"W",
+            "dims": ["N"], 
+            "description": "Maximum Power" 
+        },
+        {
+            "name": "impp",
+            "type": "float64",
+            "unit": "A",
+            "dims": ["N"],
+            "description": "Maximum power point current."
+        },
+        {
+            "name": "isc",
+            "type": "float64",
+            "unit": "A",
+            "dims": ["N"],
+            "description": "Short circuit current."
+        },
+
+        {
+            "name": "vmpp",
+            "type": "float64",
+            "unit": "V",
+            "dims": ["N"],
+            "description": "Maximum power point voltage."
+        },
+        {
+            "name": "voc",
+            "type": "float64",
+            "unit": "V",
+            "dims": ["N"],
+            "description": "Open circuit voltage."
+        }
+    ]
+}
+```
+
+## Loading an Entity with DLite
+We will now load our Entity in Python. There are several ways of doing this. 
+
+### 1. Using the json storage plugin:
+<!--[storage](link_to_storage)-->
+```python
+import dlite
+
+SolarPanelEntity = dlite.Instance.from_location('json', 'myEntity.json')
+
+```
+
+### 2. Adding the file to `storage_path`
+In this method, we append the `json` file to `storage_path` and then use the [get_instance()](https://sintef.github.io/dlite/autoapi/dlite/index.html?highlight=get_instance#dlite.get_instance) function to load the Entity using its URI. In Python, this may be done by
+
+``` python
+import dlite
+import json
+
+# Open the file and load its contents, so we can get the Entity later using its uri
+with open('myEntity.json') as entity_file:
+    data = json.load(entity_file)
+
+# Append filepath to storage path
+dlite.storage_path.append('myEntity.json')
+
+# Load Entity using its uri
+SolarPanelEntity = dlite.get_instance(data['uri']) 
```

Comment:
Wouldn't it be simpler to refer directly to the uri? That would reduce this example to

```suggestion
import dlite

# Append filepath to storage path
dlite.storage_path.append('myEntity.json')

# Load Entity using its uri
SolarPanelEntity = dlite.get_instance("http://www.ontotrans.eu/0.1/solarPanelMeasurement")
```